style.css

```diff
@@ -0,0 +1,66 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: lightgreen;
+}
+
+nav {
+  height: 10vh;
+  width: 100%;
+  background-color: aqua;
```

Comment:
So as per the goal of the project, you have to use same colors, fonts, font-sizes etc. used in the FIGMA file.
So, as per your Template the background color is `#fff`
Furthermore, it contains the background;
You can download the background like that:
![image](https://user-images.githubusercontent.com/61361037/212486708-d83ef055-ffe1-4f82-aadd-2ece5c3c84ad.png)